rel/i18n/emqx_otel_schema.hocon

```diff
@@ -1,15 +1,44 @@
 emqx_otel_schema {
 
 opentelemetry.desc: "Open Telemetry Toolkit configuration"
+opentelemetry.label: "Open Telemetry"
+
+otel_logs.desc:
+"""Open Telemetry Logs configuration. If enabled, EMQX Installs a log handler that formats events according to Open Telemetry log data model and
+exports them to the configured Open Telemetry collector or back-end."""
+otel_logs.label: "Open Telemetry Logs"
+
+otel_metrics.desc: "Open Telemetry Metrics configuration."
+otel_metrics.label: "Open Telemetry Metrics"
+
+enable.desc: "Enable or disable Open Telemetry signal."
+enable.label: "Enable."
 
 exporter.desc: "Open Telemetry Exporter"
+exporter.label: "Exporter"
+
+max_queue_size.desc:
+"""The maximum queue size. After the size is reached Open Telemetry signals are dropped."""
```

Comment:
Q: What is the strategy for dropping the signals? Does it drop the new signals until the queue is drained, or discards the queued data as well?

Reply:
It discards new signals. No need to discard the queued data, since olp mechanism generally ensures that the 'queue' stops growing once the limit is reached. The 'queue' is actually not a real queue data structure, but an ETS table. 
We keep using 'max_queue_size' because it's a term used in OpenTelemetry specification and our ETS -based implementation doesn't violate the spec. 
Each insert to ETS table is done after decrementing and getting the value of the atomic counter. Once counter is decremented to 0 (it starts with max_queue_size), the new events are drooped and the processor (gen_statem process) is notified to start a next exporting cycle. 
The actual implementation is slightly more complex since there are two ETS tables and they are reversed periodically, so that when the processor starts exporting, another table becomes available for inserts.
https://github.com/emqx/opentelemetry-erlang/blob/main/apps/opentelemetry/src/otel_batch_olp.erl#L178

Reply:
Thanks. Perhaps it's worth mentioning in the documentation?